pkg/record/recorder.go

```diff
@@ -45,20 +46,20 @@ func InitFromRecorder(recorder record.EventRecorder) {
 
 // Event constructs an event from the given information and puts it in the queue for sending.
 func Event(object runtime.Object, reason, message string) {
-	defaultRecorder.Event(object, corev1.EventTypeNormal, strings.Title(reason), message)
+	defaultRecorder.Event(object, corev1.EventTypeNormal, cases.Title(language.Und).String(reason), message)
```

Comment:
Out of curiosity, what is the `language.Und` doing?

Reply:
same question here..

Reply:
I think this is for `Undefined` language, it's not really explained in the [docs]. Using this language tag returned the expected result, as you can see in this Go Playground example: https://go.dev/play/p/fNew-KrewPY

Reply:
Makes sense. Maybe change it to `language.English` then?

Reply:
I think that makes sense, considering the reasons should be in english so all consumers of them can understand them.